src/styles/card.scss

```diff
@@ -0,0 +1,143 @@
+$image_width: 198px;
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $image_width;

```

Comment:
is this really an image width?
add border-box to avoid weird numbers, 198 -> 200